examples/image-streamer/build_plan.rb

```diff
@@ -9,7 +9,14 @@
 # CONDITIONS OF ANY KIND, either express or implied. See the License for the
 # specific language governing permissions and limitations under the License.
 
-require_relative '../../_client_i3s' # Gives access to @client
+require_relative '../_client_i3s' # Gives access to @client
+# Supported APIs:
+# - 300, 500, 600
+
+# Resources that can be created according to parameters:
+# api_version = 300 & variant = Synergy to OneviewSDK::ImageStreamer::API300::BuildPlan
+# api_version = 500 & variant = Synergy to OneviewSDK::ImageStreamer::API500::BuildPlan
+# api_version = 600 & variant = Synergy to OneviewSDK::ImageStreamer::API600::BuildPlan
 
 # Example: Create a build plan for an API300 Image Streamer
 # NOTE: This will create three build plans with the following names 'Build_Plan_1', 'Build_Plan_2' and 'Build_Plan_3', then delete them.
```

Comment:
This comment needs to be changes as you are not using the build plan names mentioned above

Reply:
Can ignore this,